scripts/create_tx_start_stop_bed.R

```diff
@@ -0,0 +1,199 @@
+#'
+#' Parse a gtf file into objects for easy feature extraction
+#' https://www.ncbi.nlm.nih.gov/data-hub/genome/GCF_009914755.1/
```

Comment:
Some variation of argparse would be useful here, e.g. 

https://blog.sellorm.com/2017/12/30/command-line-utilities-in-r-pt-6/
https://cran.r-project.org/web/packages/argparse/index.html

arguments: input, output

Reply:
I'm using optparse -- are you saying you'd prefer argparse?

Reply:
This is what running looks like:

```
➜  isocomp git:(extract_tx_regions) scripts/create_tx_start_stop_bed.R --help
Usage: scripts/create_tx_start_stop_bed.R [options]


Options:
	-g GTF, --gtf=GTF
		path to the gtf annotation set. Suggested source:  https://www.ncbi.nlm.nih.gov/data-hub/genome/GCF_009914755.1/

	-t TXDB, --txdb=TXDB
		path to a bioconductor sqlite TxDb. If one DNE, it will be created. In that case, provide a writeable path.

	-h, --help
		Show this help message and exit
```

I would suggest not adding the GTF input since its available from NCBI through that link -- thoughts?

A run command looks like this:

```
scripts/create_tx_start_stop_bed.R -g data/GCF_009914755.1/genomic.sorted.gtf -t data/t2t_txdb.sqlite
```
I suppose I should put that into the add_scripts.md